platform/android/java/editor/src/debug/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="godot_editor_name_string">Godot Editor (debug)</string>
```

Comment:
This is used by the `target=release_debug` build, which is the "production" build target for the editor. So it's a bit weird to have the installed application be called `Godot Editor (debug)`, it should be just `Godot Editor` and `(debug)` should be added only for `target=debug` builds.